<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FplGui_Faulttree_Gate_Wid</class>
 <widget class="QWidget" name="FplGui_Faulttree_Gate_Wid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>772</width>
        <height>668</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>690</width>
           <height>571</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="Line" name="line_6">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>97</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>377</y>
            <width>90</width>
            <height>96</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Overflow</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_overflow">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>It can not be changed!</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
               <normaloff>:/gate_overflow_icon</normaloff>:/gate_overflow_icon</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Line" name="line_18">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>472</y>
            <width>20</width>
            <height>16</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>17</y>
            <width>381</width>
            <height>41</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>10</y>
             <width>201</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Failure event gate-type</string>
           </property>
          </widget>
         </widget>
         <widget class="Line" name="line_17">
          <property name="geometry">
           <rect>
            <x>285</x>
            <y>122</y>
            <width>20</width>
            <height>85</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="Line" name="line_5">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>57</y>
            <width>20</width>
            <height>51</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_5">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>207</y>
            <width>109</width>
            <height>135</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>
Stability 
landward side
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_ero_landward">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>It can not be changed!</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>45</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>67</y>
            <width>191</width>
            <height>491</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Hydraulic </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="Line" name="line_9">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>340</y>
            <width>20</width>
            <height>38</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_8">
          <property name="geometry">
           <rect>
            <x>19</x>
            <y>487</y>
            <width>361</width>
            <height>55</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Wind-induced wave</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_wind_wave">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Line" name="line_11">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>122</y>
            <width>20</width>
            <height>85</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_12">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>87</y>
            <width>33</width>
            <height>36</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>or</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_2">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>87</y>
            <width>33</width>
            <height>36</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>or</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>215</x>
            <y>67</y>
            <width>181</width>
            <height>491</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Structurell</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <widget class="Line" name="line_19">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>273</y>
             <width>20</width>
             <height>148</height>
            </rect>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_16">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>207</y>
            <width>125</width>
            <height>135</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Structurell stability 
of the gate</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_structure">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
               <normaloff>:/gate_struct_icon</normaloff>:/gate_struct_icon</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_6">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>377</y>
            <width>90</width>
            <height>96</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Overtopping</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_overtopping">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
               <normaloff>:/gate_overtop_icon</normaloff>:/gate_overtop_icon</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Line" name="line_14">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>340</y>
            <width>20</width>
            <height>38</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_3">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>17</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>231</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Not-close event gate-type</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_failure_9">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>90</y>
            <width>109</width>
            <height>135</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Technical 
or human
failure</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_not_close">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>It can not be changed!</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
               <normaloff>:/gate_notcloseicon</normaloff>:/gate_notcloseicon</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Line" name="line_20">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>57</y>
            <width>20</width>
            <height>35</height>
           </rect>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <zorder>groupBox_3</zorder>
         <zorder>groupBox</zorder>
         <zorder>line_6</zorder>
         <zorder>groupBox_failure_7</zorder>
         <zorder>line_18</zorder>
         <zorder>groupBox_failure</zorder>
         <zorder>line_17</zorder>
         <zorder>line_5</zorder>
         <zorder>groupBox_failure_5</zorder>
         <zorder>line_9</zorder>
         <zorder>groupBox_failure_8</zorder>
         <zorder>line_11</zorder>
         <zorder>groupBox_failure_12</zorder>
         <zorder>groupBox_failure_2</zorder>
         <zorder>groupBox_failure_16</zorder>
         <zorder>groupBox_failure_6</zorder>
         <zorder>line_14</zorder>
         <zorder>groupBox_failure_3</zorder>
         <zorder>groupBox_failure_9</zorder>
         <zorder>line_20</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc"/>
  <include location="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc"/>
  <include location="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
